app/liquid_tags/comment_tag.rb

```diff
@@ -14,9 +14,7 @@ def render(_context)
   end
 
   def find_comment(id_code)
-    Comment.find(id_code.to_i(26))
-  rescue ActiveRecord::RecordNotFound
-    raise StandardError, "Invalid comment ID or comment does not exist"
+    Comment.find_by(id_code: id_code)
```

Comment:
I have a question about how I am finding the Comment here.

Originally, the code was
```ruby
Comment.find(id_code.to_i(26))
```

Please advise me if the conversion of the `id_code` to Integer Base 26 is essential. Cos that would mean my approach above is inappropriate 🤷‍♀️

Reply:
I think it's fine, as `id_code` is the serialization of `id.to_i(26)` as indicated here 

https://github.com/forem/forem/blob/53deac75cde12bca16f680369424dfd187705f85/app/models/comment.rb#L90-L94

If the liquid tag tests are still working then it's working ;)